### PR TITLE
[ja] Translate tasks/administer-cluster/manage-resources/memory-default-namespace/ into Japanese 

### LINKED ISSUE #31427: [ja] Translate tasks/administer-cluster/manage-resources/memory-default-namespace/ into Japanese
# This is a Feature Request

## What would you like to be added

Translate [tasks/administer-cluster/manage-resources/memory-default-namespace/(https://kubernetes.io/docs/tasks/administer-cluster/manage-resources/memory-default-namespace/)  into Japanese.

## Why is this needed

This page has not been translated.

## Comments

/language ja
/assign

### PR DESCRIPTION
**Pull request by ptux**
resolve #31427

### CODE REVIEW

File: content/ja/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+```
+
+## デフォルトのメモリ制限と要求の動機
+
+ネームスペースにリソースクォータがある場合、メモリ制限のデフォルト値を設定しておくと便利です。
+
+以下はリソースクォータがネームスペースに課す制限のうちの2つです。
+
+* ネームスペースで実行されるすべてのコンテナは、独自のメモリ制限を持つ必要があります。
+* ネームスペース内のすべてのコンテナで使用されるメモリの総量は、指定された制限を越えてはなりません。
```
Comment (t-inu) [inline content/ja/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md:150]:
```suggestion
* ネームスペース内のすべてのコンテナで使用されるメモリの総量は、指定された制限を超えてはなりません。
```
In this case, I think "超える" is usually used.

File: content/ja/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+```shell
+kubectl delete namespace default-mem-example
+```
+
+
+
+## {{% heading "whatsnext" %}}
+
+
+### クラスター管理者のため
```
Comment (t-inu) [inline content/ja/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md:167]:
```suggestion
### クラスター管理者向け
```
How about "○○向け" ?

File: content/ja/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+
+* [Configure Minimum and Maximum CPU Constraints for a Namespace](/docs/tasks/administer-cluster/manage-resources/cpu-constraint-namespace/)
+
+* [Configure Memory and CPU Quotas for a Namespace](/docs/tasks/administer-cluster/manage-resources/quota-memory-cpu-namespace/)
+
+* [Configure a Pod Quota for a Namespace](/docs/tasks/administer-cluster/manage-resources/quota-pod-namespace/)
+
+* [Configure Quotas for API Objects](/docs/tasks/administer-cluster/quota-api-object/)
+
+### アプリケーション開発者のため
```
Comment (t-inu) [inline content/ja/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md:181]:
```suggestion
### アプリケーション開発者向け
```
ditto.

**Review by t-inu — COMMENTED** [review]
It seems the en page has been largely updated after you translated it.
I reviewed the whole page and suggested about the updated text. I hope it's your help.
Please refer to the [history](https://github.com/kubernetes/website/commits/main/content/en/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md) for diffs.

File: content/ja/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+weight: 10
+description: >-
+  ネームスペースのデフォルトのメモリリソース制限を定義して、そのネームスペース内のすべての新しいPodにメモリリソース制限が設定されるようにします。
+---
+
+<!-- overview -->
+
+このページでは、{{< glossary_tooltip text="ネームスペース" term_id="namespace" >}}のデフォルトのメモリ要求と制限を設定する方法を説明します。
+
+Kubernetesクラスターはネームスペースに分割することができます。デフォルトのメモリ[制限](ja/docs/concepts/configuration/manage-resources-containers/#%E8%A6%81%E6%B1%82%E3%81%A8%E5%88%B6%E9%99%90)を持つネームスペースがあり、独自のメモリ制限を指定しないコンテナでPodを作成しようとすると、{{< glossary_tooltip text="コントロールプレーン" term_id="control-plane" >}}はそのコンテナにデフォルトのメモリ制限を割り当てます。
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md:13]:
```suggestion
Kubernetesクラスターはネームスペースに分割することができます。デフォルトのメモリ[制限](/ja/docs/concepts/configuration/manage-resources-containers/#%E8%A6%81%E6%B1%82%E3%81%A8%E5%88%B6%E9%99%90)を持つネームスペースがあり、独自のメモリ制限を指定しないコンテナでPodを作成しようとすると、{{< glossary_tooltip text="コントロールプレーン" term_id="control-plane" >}}はそのコンテナにデフォルトのメモリ制限を割り当てます。
```

and can you add an English ID for the link target page, and use it?

File: content/ja/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md [hunk 5]
```diff
@@ -0,0 +1,199 @@
+---
+title: ネームスペースのデフォルトのメモリ要求と制限を設定する
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md:2]:
```suggestion
title: ネームスペースのデフォルトのメモリー要求と制限を設定する
```
以下同様にお願いします

File: content/ja/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md [hunk 6]
```diff
@@ -0,0 +1,199 @@
...
+    memory: 128Mi
+```
+
+## デフォルトのメモリ制限と要求の動機
+
+ネームスペースにメモリ{{< glossary_tooltip text="リソースクォータ" term_id="resource-quota" >}}が設定されている場合、メモリ制限のデフォルト値を設定しておくと便利です。
+
+以下はリソースクォータがネームスペースに課す制限のうちの2つです。
+
+* ネームスペースで実行されるすべてのPodについて、Podとその各コンテナにメモリ制限を設ける必要があります。(Pod内のすべてのコンテナに対してメモリ制限を指定すると、Kubernetesはそのコンテナの制限を合計することでPodレベルのメモリ制限を推測することができます)。
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md:154]:
```suggestion
* ネームスペースで実行されるすべてのPodについて、Podとその各コンテナにメモリ制限を設ける必要があります。(Pod内のすべてのコンテナに対してメモリ制限を指定すると、Kubernetesはそのコンテナの制限を合計することでPodレベルのメモリ制限を推測することができます。)
```

or

```suggestion
* ネームスペースで実行されるすべてのPodについて、Podとその各コンテナにメモリ制限を設ける必要があります(Pod内のすべてのコンテナに対してメモリ制限を指定すると、Kubernetesはそのコンテナの制限を合計することでPodレベルのメモリ制限を推測することができます)。
```

File: content/ja/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md [hunk 7]
```diff
@@ -0,0 +1,197 @@
...
+
+
+## {{% heading "whatsnext" %}}
+
+
+### クラスター管理者向け
+
+* [Configure Default CPU Requests and Limits for a Namespace](/docs/tasks/administer-cluster/manage-resources/cpu-default-namespace/)
+
+* [Configure Minimum and Maximum Memory Constraints for a Namespace](/docs/tasks/administer-cluster/manage-resources/memory-constraint-namespace/)
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md:181]: It seems this page has been translated. Please update the link text and url

File: content/ja/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md [hunk 8]
```diff
@@ -0,0 +1,197 @@
...
+
+* [Configure Memory and CPU Quotas for a Namespace](/docs/tasks/administer-cluster/manage-resources/quota-memory-cpu-namespace/)
+
+* [Configure a Pod Quota for a Namespace](/docs/tasks/administer-cluster/manage-resources/quota-pod-namespace/)
+
+* [Configure Quotas for API Objects](/docs/tasks/administer-cluster/quota-api-object/)
+
+### アプリケーション開発者向け
+
+* [Assign Memory Resources to Containers and Pods](/docs/tasks/configure-pod-container/assign-memory-resource/)
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md:193]: ditto

File: content/ja/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md [hunk 9]
```diff
@@ -0,0 +1,197 @@
...
+
+* [Assign CPU Resources to Containers and Pods](/docs/tasks/configure-pod-container/assign-cpu-resource/)
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md:195]: ditto

File: content/ja/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md [hunk 10]
```diff
@@ -0,0 +1,197 @@
...
+
+* [Configure Quality of Service for Pods](/docs/tasks/configure-pod-container/quality-service-pod/)
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md:197]: ditto

File: content/ja/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md (hunk 4 above)
Comment (t-inu) [reply content/ja/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md]: > and can you add an English ID for the link target page, and use it?

I created the PR https://github.com/kubernetes/website/pull/33213. Please review.

File: content/ja/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md [hunk 11]
```diff
@@ -0,0 +1,197 @@
...
+weight: 10
+description: >-
+  ネームスペースのデフォルトのメモリーリソース制限を定義して、そのネームスペース内のすべての新しいPodにメモリーリソース制限が設定されるようにします。
+---
+
+<!-- overview -->
+
+このページでは、{{< glossary_tooltip text="ネームスペース" term_id="namespace" >}}のデフォルトのメモリー要求と制限を設定する方法を説明します。
+
+Kubernetesクラスターはネームスペースに分割することができます。デフォルトのメモリー[制限](/ja/docs/concepts/configuration/manage-resources-containers/#%E8%A6%81%E6%B1%82%E3%81%A8%E5%88%B6%E9%99%90)を持つネームスペースがあり、独自のメモリー制限を指定しないコンテナでPodを作成しようとすると、{{< glossary_tooltip text="コントロールプレーン" term_id="control-plane" >}}はそのコンテナにデフォルトのメモリー制限を割り当てます。
```
Comment (t-inu) [inline content/ja/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md:13]:
```suggestion
Kubernetesクラスターはネームスペースに分割することができます。デフォルトのメモリー[制限](/ja/docs/concepts/configuration/manage-resources-containers/#requests-and-limits)を持つネームスペースがあり、独自のメモリー制限を指定しないコンテナでPodを作成しようとすると、{{< glossary_tooltip text="コントロールプレーン" term_id="control-plane" >}}はそのコンテナにデフォルトのメモリー制限を割り当てます。
```
https://github.com/kubernetes/website/pull/33213 was marged.
Now #requests-and-limits is available. Please change the link.

Comment (t-inu) [reply content/ja/docs/tasks/administer-cluster/manage-resources/memory-default-namespace.md:13]: @ptux Could you please change the link?

**Pull request merged**